__mocks__/extensionizer.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Brilliant! We are . seriously lacking in tests overall (i.e. we have . none), this is great to make a start.

Just add a copyright header to this file as well.